apps/base/utils.py

```diff
@@ -32,3 +33,14 @@ def __call__(self, instance, filename):
         filename = '{}{}'.format(uuid.uuid4(), extension)
         filename = os.path.join(self.path, filename)
         return filename
+
+
+def get_model_object(model_name):
+    def get_model_by_pk(pk):
+        try:
+            model_object = model_name.objects.get(pk=pk)
+            return model_object
+        except model_name.DoesNotExist:
+            raise NotFound('{} {} does not exist'.format(model_name.__name__, pk))
+    get_model_by_pk.__name__ = "get_{}_object".format(model_name)
```

Comment:
One can pass model name like ```Challenge```, so when creating function, we should use ```model_name.lower()```. Also check if we should use `model_name`or `model_name.__name__`. I think name one should be used.